oaknational/Oak-Web-Application
# Feat/lesq 435/share card container [LESQ-435]

## Description

- Create component and story for share card group
- Refactor out some shared logic between download and share cards 
- Rename prop on resource card to be more generic between download/share

## How to test

1. Go to https://deploy-preview-2085--owa-storybook.netlify.thenational.academy/?path=/docs/download-components-share-card-group--docs
2. Click on download components / share card group
3. You should see stories for a few different resources present

## Screenshots

How it should now look:
<img width="500" alt="Screenshot 2023-11-09 at 09 04 51" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/f51e5d0f-bf5f-4eab-9f7c-9a8c41bcf637">

## Checklist

- [ ] Should match [figma design](https://www.figma.com/file/LSjbzdvHo17XqXGBRNJjQP/Old-%2F-new-content?node-id=4468%3A27528&mode=dev)
